docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,286 @@
+  comments, which the pull request author is encouraged to address,
+  but alternatively can create an issue with the suggested changes,
+  which will be addressed in a subsequent pull request by the author.
+- Assigned reviewers should submit their review in as timely a manner
+  as possible.
+- If any requested change is unclear to the author, they should ping
+  the reviewer and ask for clarification.  If needed, the author and
+  reviewers should talk to one another (in person, via Google hangout,
+  ...) to resolve any issues.
+- Reviewers are encouraged to ping others to get opinions on code they
+  are unsure about.
+- It is permissible to have a group code review led by one of the
+  reviewers. The reviewer should comment on who was present at the
+  group review.
+- If necessary, pull requests can also be discussed at one of the
+  weekly SpECTRE meetings.
+- If changes are requested, the author should fix all of them in one
+  or more fixup commits, and then ping the reviewers that the pull
+  request is updated.  In addition the `updated` label can be added.
+- Once a pull request is updated, the reviewers should either request
+  further changes (removing the `updated` label) or tell the pull
+  request author to squash their commits.
+- Once the commits are squashed, the author should ping the reviewers.
+- Once all reviewers have approved the pull request, someone should
+  ping the `@sxs-collaboration/spectre-core-devs`.
+- One of the core developers will perform a final cursory review,
+  requesting changes only for major problems, and commenting on other
+  possible changes.
+- The pull request author should address all final requested changes
+  and may either also fix final suggested changes, or create an issue
+  with the suggested changes, which will be addressed in a subsequent
+  pull request by the author.
+- The SpECTRE core developer will merge the pull request when
+  satisfied.
+
+Critical bug fixes (i.e. the code is broken) can be merged after two
+expedited reviews by SpECTRE core developers.  If necessary, an issue
+can be created if further changes are desired.
+
+Pull requests that are designated `new design` are expected to have a
+longer review period, including discussions during weekly SpECTRE
+meetings.  SpECTRE core developers will provide reasonable review
+deadlines once the new design is finalized.
+
+If you would like feedback on a pull request prior to it being ready
+for formal reveiw, please label it with `in progress` and ping
+whomever you wish to get feedback from.  As long as the `in progress`
+label remains, no one is obligated to review the pull request.
+
+
+### Git Commit Messages
+
+* Use the present tense ("Add feature" not "Added feature")
+* Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
+* Limit the first line to 72 characters or less
+* If needed, a blank second line followed by a more complete description
```

Comment:
> If needed, add ..., also limited to 72 characters per line

Reply:
I think GitHub expects commit messages to be limited to 72 chars, at least it does some wrapping. Not sure, I can try to find out.

Reply:
> The reason for wrapping your description lines at the 72nd mark is that git log adds a padding of 4 blank spaces when displaying the commit message. To center the message perfectly, and make it read well on an 80-column terminal, you’d want to leave space for 4 more blank spaces at right side. That’s where 72 comes from.

https://medium.com/@preslavrachev/what-s-with-the-50-72-rule-8a906f61f09c